lib/siwapp/schema/customers.ex

```diff
@@ -0,0 +1,59 @@
+defmodule Siwapp.Schema.Customers do
+  use Ecto.Schema
+  alias Siwapp.Schema.Invoices
+  import Ecto.Changeset
+
+  schema "customers" do
+    field :identification, :string
+    field :name, :string
+    field :email, :string
+    field :contact_person, :string
+    field :active, :boolean, default: true
+    field :deleted_at, :utc_datetime
+    field :invoicing_address, :string
+    field :shipping_address, :string
+    field :meta_attributes, :map
+    has_many :invoices, Invoices
+
+    timestamps()
+  end
+
+  @doc false
+  def changeset(customers, attrs) do
+    customers
+    |> cast(attrs, [
```

Comment:
Maybe instead of adding all the costumer keys as a list at the cast function, you could only add the keys you need. In this case, it must appear :name or :indentification and the other keys are optional

Reply:
It appears to be the same in all the schema/*.ex